src/tests/Microsoft.PowerFx.Core.Tests/ExpressionTestCases/Boolean.txt

```diff
@@ -8,22 +8,28 @@ false
 #Error
 
 >> Boolean(" true")
-true
+#Error
```

Comment:
(Kind=InvalidArgument). Same for the other (runtime) errors when converting from string

Reply:
done